src/dali/plugin/input/rest/binance_com.py

```diff
@@ -186,11 +186,18 @@ def _initialize_client(self) -> binance:
             }
         )
 
+    @property
+    def _client(self) -> binance:
```

Comment:
Not sure about changing the return type: I guess if mypy doesn't complain I'm good with it (the other option would have been to leave the return type as `Exchange` and use `cast` in the call sites). Otherwise it's fine to override the property because you're adding extra runtime checks.